CMakeLists.txt

```diff
@@ -294,8 +294,8 @@ else()
 endif()
 
 if(DEFINED REF_BLAS_ROOT)
-  find_file(REF_BLAS_LIBNAME NAMES blas.dll libblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
-  find_file(REF_CBLAS_LIBNAME NAMES cblas.dll libcblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
+  find_file(ONEAPI_ONEMKL_REF_BLAS_LIBNAME NAMES blas.dll libblas.so HINTS ${REF_BLAS_ROOT} PATH_SUFFIXES lib lib64)
```

Comment:
Overall the change looks okay to me, my only concern is about the macro prefix choice. ONEAPI_ONEMKL is a little bit redundant and heavy in my opinion, could we consider using only ONEMKL or follow the namespace with ONEAPI_MKL (this one would have my preference)?

Reply:
I'm not sure we could use `ONEAPI_MKL` as I vaguely remember we had to remove occurrences of "MKL" in the past. The exceptions are the name of the backends MKLCPU, MKLGPU and the mkl namespace.
I don't mind too much between `ONEAPI_ONEMKL` or `ONEMKL`.

Reply:
Thank you for your review @lhuot. 
I agree it is quite verbose. In my opinion, the best option is to use `ONEMKL` for now, so that it can be renamed later in `ONEMATH`. What do you think?

Reply:
I also support shorter name if possible (longer macro name increase the chance of a typo), `ONEMKL` looks great to me while we are migrating to oneMath, but does it make sense to start changing it to `ONEMATH` already to reduce users efforts for migrating to new macro names in their build configurations?

Reply:
thank you for the feedback @mkrainiuk,
I don't like the idea of having these variables already renamed while everything else isn't. I know @Rbiessy  is working on it, but I don't want to have this period with different names in between his renaming patch and this one. Moreover, these variables change should not affect users but only internal headers. So I would rather use `ONEMKL` for now.